### ISSUE TITLE
Minimal catalog validation

### ISSUE THREAD

**Issue by grugnog** [GSA/open-product-accessibility-template#10]

**Comment by dmundra** [comment]
Rough notes from discussion with @grugnog after reviewing PR #57.
* The chapters will need keys because we can use that to match/bring in WCAG criteria from the catalog.
* The user should be able to specify different catalogs and generate outputs.
* We will be merging the catalog and user input when generating the output in markdown/HTML.

**Comment by dmundra** [comment]
Submitted PR ☝🏾  with the catalog validation solution. Wanted to note here that the WCAG JSONs have a lot of data but they are driving different tools so I made a basic catalog from scratch that drives our validation logic. I think we need that freedom right now.

Another thought I had is that in this or next sprint I need help with QAing the CLI so that we are kicking the tires on the current route and someone is writing YAML inputs and catalogs that we can verify whether it works or not.

**Comment by dmundra** [comment]
Created #60 to QA in the next sprint.
